config/default.yaml

```diff
@@ -105,9 +109,9 @@ dspace::java_version : '7'
 # Always build/compile DSpace using "vagrant.properties"
 # instead of the normal "build.properties" file
```

Comment:
Remove these comments...they are no longer valid on "master". vagrant.properties and build.properties are retired